frontend/package.json

```diff
@@ -14,7 +14,7 @@
     "web-vitals": "^2.1.4"
   },
   "scripts": {
-    "start": "react-scripts --openssl-legacy-provider start",
+    "start": "react-scripts start --openssl-legacy-provider",
```

Comment:
Is this something that you did? Or something that happened as a result of something else? If it was you, what does it do?